.github/workflows/container-image.yaml

```diff
@@ -8,9 +8,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v1
-    - name: Build container image
-      run: |
-        docker build -t aws-efs-csi-driver .
```

Comment:
No need for a separate build step because `docker buildx` build and push are run at the same step.

Reply:
> The docker buildx build subcommand has a number of flags which determine where the final image will be stored. By default, i.e. if none of the flags are specified, the resulting image will remain captive in docker’s internal build cache. This is unlike the regular docker build command which stores the resulting image in the local docker images list. 

> --push: This flag tells docker to push the resulting image to a docker registry. This currently is the best way to store multi-architecture images.

https://medium.com/@artur.klauser/building-multi-architecture-docker-images-with-buildx-27d80f7e2408